WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationCommentDetailViewController.swift

```diff
@@ -229,16 +232,39 @@ private extension NotificationCommentDetailViewController {
 
                 self.fetchComment(self.commentID, completion: { [weak self] comment in
                     guard let comment = comment else {
-                        self?.showErrorView()
+                        self?.showErrorView(title: NoResults.errorTitle, subtitle: NoResults.errorSubtitle)
                         return
                     }
 
                     self?.comment = comment
+                }, failure: { [weak self] error in
+                    guard let self = self else {
+                        return
+                    }
+                    self.showErrorView(error: error)
                 })
+            }, failure: { [weak self] error in
+                guard let self = self else {
+                    return
+                }
+                self.showErrorView(error: error)
```

Comment:
Nit: In the nested closures, we currently redeclare `[weak self]` after it has already been captured as `weak` by an outer closure. For conciseness and based on [this Stackoverflow question](https://stackoverflow.com/questions/38739129/do-capture-lists-of-inner-closures-need-to-redeclare-self-as-weak-or-unowne), we can safely omit the redundant `weak` capture in the inner closures while still avoiding strong reference cycles.